code/glTF2Asset.h

```diff
@@ -670,7 +669,12 @@ namespace glTF2
         inline uint8_t* StealData();
 
         inline void SetData(uint8_t* data, size_t length, Asset& r);
-    };
+	};
+
+	static const vec4 defaultBaseColor = {1, 1, 1, 1};
+	static const vec3 defaultEmissiveFactor = {0, 0, 0};
+	static const vec4 defaultDiffuseFactor = {1, 1, 1, 1};
+	static const vec3 defaultSpecularFactor = {1, 1, 1};
```

Comment:
Whoops, mislead you about the `static` thing. `static` has 2 separate meanings in C++: if you declare a member of a `class` as `static`, it'll be allocated once during the lifetime of the program. However, if you declare a member of a `namespace` as static, it tells the compiler that that variable should use "internal linkage", meaning it's only available inside the current compilation unit (usually a .cpp file). Since these defaults are at the namespace level, it's sufficient to declare them as `const vec3`, etc. (i.e. without the `static` qualifier).

Reply:
Ok!